doc/source/about.md

```diff
@@ -0,0 +1,38 @@
+# About GeoPandas
+
+```{toctree}
+:maxdepth: 2
+:caption: About
+:hidden:
+
+Roadmap <about/roadmap>
+Team <about/team>
+Citing <about/citing>
+Logo <about/logo>
+```
+
+
+GeoPandas is a project to add support for geographic data to pandas objects.
+It currently implements `GeoSeries` and `GeoDataFrame` types which are subclasses
+of `pandas.Series` and `pandas.DataFrame` respectively. GeoPandas objects can act
+on `shapely` geometry objects and perform geometric operations.
+
+```{container} button
+
+{doc}`Project Roadmap <about/roadmap>` {doc}`Team <about/team>`
+{doc}`Citing <about/citing>` {doc}`Logo <about/logo>`
+```
+
+## Project history
+
+Kelsey Jordahl founded GeoPandas project in 2013 and released a version 0.1.0 in July 2014. In 2016, Joris Van den Bossche took the lead and became the maintainer of the project. Since the beginning, GeoPandas is a BSD-licensed open-source project supported by a [community of contributors](https://github.com/geopandas/geopandas/graphs/contributors) from around the world.
```

Comment:
@jorisvandenbossche can you check if the dates are correct? I am not sure when you became the maintainer.

Reply:
Checking my emails, it was indeed 2016 (June 2016, 0.2 release was the first I was involved with). 

Compared to 2016, we are with a bigger team now, so maybe we should mention something like "and is now maintained by a team of core developers" (linking to the team page)